xmtp_mls/Cargo.toml

```diff
@@ -38,6 +38,8 @@ openmls_rust_crypto = { workspace = true }
 openmls_traits = { workspace = true }
 prost = { workspace = true, features = ["prost-derive"] }
 rand = { workspace = true }
+reqwest = { version = "0.12.4", features = ["stream"] }
```

Comment:
maybe not something to do in the future, but since the message history stuff is only used by the message-history crate (or anybody else impl message history) what do you think of feature-gating `message-history` in order to reduce the resulting WASM binary for stuff that doesn't need message history